### DISCUSSION TITLE
pandoc install fails on mac m1

### DISCUSSION THREAD

**Discussion by maxodoble** [Homebrew/discussions#1652] [Everyday usage] [answered]
Hi,

tried to install pandoc today:

`brew install pandoc`

and it fails with this error:

`Pouring pandoc--2.14.0.1.arm64_big_sur.bottle.tar.gz Error: Couldn't find manifest matching bottle checksum.`

complete error message:

`brew install pandoc
==> Downloading https://ghcr.io/v2/homebrew/core/pandoc/manifests/2.14.0.1
Already downloaded: /Users/hra/Library/Caches/Homebrew/downloads/fddf05ee033ff9d88801ab7d840793b9e87becb6e7f49b40705408b7fa9611cc--pandoc-2.14.0.1.bottle_manifest.json
==> Downloading https://ghcr.io/v2/homebrew/core/pandoc/blobs/sha256:c74ca2fe9c631cc0018691cd5501577e8b42fb26574230aa41847a6b92307c92
Already downloaded: /Users/hra/Library/Caches/Homebrew/downloads/782a735832ed846edacd89a9f1c0a3f01fee4f5e42f651c6c741b25da6b6991a--pandoc--2.14.0.1.arm64_big_sur.bottle.tar.gz
==> Pouring pandoc--2.14.0.1.arm64_big_sur.bottle.tar.gz
Error: Couldn't find manifest matching bottle checksum.
/opt/homebrew/Library/Homebrew/software_spec.rb:395:in `tab_attributes'
/opt/homebrew/Library/Homebrew/formula.rb:2287:in `bottle_tab_attributes'
/opt/homebrew/Library/Homebrew/utils/bottles.rb:100:in `load_tab'
/opt/homebrew/Library/Homebrew/formula_installer.rb:1150:in `pour'
/opt/homebrew/Library/Homebrew/formula_installer.rb:418:in `install'
/opt/homebrew/Library/Homebrew/cmd/install.rb:422:in `install_formula'
/opt/homebrew/Library/Homebrew/cmd/install.rb:324:in `block in install'
/opt/homebrew/Library/Homebrew/cmd/install.rb:322:in `each'
/opt/homebrew/Library/Homebrew/cmd/install.rb:322:in `install'
/opt/homebrew/Library/Homebrew/brew.rb:122:in `<main>'`

what can i do to get pandoc installed?

thanks, 
max

**Comment by SMillerDev** [comment]
> what can i do to get pandoc installed?

Use a homebrew install on x86 (through Rosetta in your case) because pandoc (and its dependencies) don't work on ARM

**Comment by cho-m** [comment, marked as answer]
You may want to try removing the `pandoc` manifest files from your cache (either using `brew` commands or manually):
- `/Users/hra/Library/Caches/Homebrew/`
- `/Users/hra/Library/Caches/Homebrew/downloads/`

The ARM bottle was only added today https://github.com/Homebrew/homebrew-core/commit/3fea79a8f8bdf8a14c12b2d9a8d3bbe5b507c463 but it shares the same version number, so if you had previously obtained a copy of v2.14.0.1 manifest (perhaps via Rosetta Homebrew), then it may be outdated.

---

As datapoint, I had no issues installing here:
```console
❯ uname -m
arm64

❯ brew install pandoc
==> Downloading https://ghcr.io/v2/homebrew/core/pandoc/manifests/2.14.0.1
######################################################################## 100.0%
==> Downloading https://ghcr.io/v2/homebrew/core/pandoc/blobs/sha256:c74ca2fe9c631cc0018691cd5501577e8b42fb26574230aa41847a6b923
==> Downloading from https://pkg-containers.githubusercontent.com/ghcr1/blobs/sha256:c74ca2fe9c631cc0018691cd5501577e8b42fb26574
######################################################################## 100.0%
==> Pouring pandoc--2.14.0.1.arm64_big_sur.bottle.tar.gz
🍺  /opt/homebrew/Cellar/pandoc/2.14.0.1: 10 files, 227.6MB

❯ file /opt/homebrew/bin/pandoc
/opt/homebrew/bin/pandoc: Mach-O 64-bit executable arm64
```

**Reply by maxodoble** [reply]
Thanks,

indeed I had an existing manifest from the rosetta brew pandoc install. After removing this manifest pandoc installed without error and is running now natively on mac arm brew, and i can finally get rid of the rosetta brew installation, because all packages I care about are running natively now!

Thanks again
